### ISSUE TITLE
Did this ever work?

### ISSUE THREAD

**Issue by samuk** [luandro/meshpi#4]
It's unclear from the Readme if this actually works or not?

I'm deploying Pi's anyway and was going to add mesh capability as a nice to have.

It would be great if they were compatible with Libremesh, or am I better off just going for a subnodes/batman-adv install?

**Comment by luandro** [comment]
Nope, sorry never got this to work. Gave up and started working with routers as it made more sense for my case.
